CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing
+
+This document provides guidelines for contributing to samples.
+
+## Dependencies
+
+The following dependencies must be installed on the development system:
+
+- [Docker Engine][docker-engine]
+- [Google Cloud SDK][google-cloud-sdk]
+- [make]
+
+## Integration Testing
+
+Integration tests are used to verify that the samples are actuatable 
+by Terraform. Tests are dynamically discovered based on directories in
+repo root and executed using the [blueprint-test](https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test) framework.
+
+### Test Environment
+
+The easiest way to test the samples are in isolated test projects. The
+setup for projects are defined in [test/setup](./test/setup/)
+directory.
+
+To use this setup, you need to authenticate gcloud with an identity that has
+Project Creator access on a folder and the Billing Account User role on a billing account.
+
+Set the following environment variables.
+```
+export TF_VAR_org_id="your_org_id"
```

Comment:
I set these to the project info that I typically use for testing:

```
 export TF_VAR_org_id="compute-writers.joonix.net"
 export TF_VAR_folder_id="individual-projects"
 export TF_VAR_billing_account="01B783-CE7C44-CAFEF8"
```

Should I be setting them to something else?

Reply:
The `TF_VAR_folder_id` should be just the numeric id. Other than that, it looks good. The other issue maybe permissions as I mentioned below.

Reply:
Yeah it looks like I'm not a billing admin. I'm trying to get that fixed, but in the meantime maybe another reviewer can test the procedure so I don't hole it up.